### ISSUE TITLE
VoiceOver does not read Publish to MS/Audio windows or export window from Publish tab

### ISSUE THREAD

**Issue by zacjansheski** [musescore/MuseScore#19954]
### Issue type

UI bug

### Bug description

VoiceOver does not read these windows after selecting while in the publish tab

1. Publish to MuseScore.com
2. Share on Audio.com
3. Export

### Steps to reproduce

While using a screen reader, from the Publish tab open the windows listed above.

### Screenshots/Screen recordings

https://github.com/musescore/MuseScore/assets/69917893/0b4ab943-1c1f-4319-a66d-85f1ada02b15

### MuseScore Version

OS: macOS 14.0, Arch.: x86_64, MuseScore version (64-bit): 4.2.0-233100302, revision: github-musescore-musescore-59e727b

### Regression

Yes, this used to work in a previous version of MuseScore 4.x

### Operating system

Mac OS 14

### Additional context

Windows read as desired from File menu

This is associated with this new share on audio.com PR

https://github.com/musescore/MuseScore/pull/19826

**Comment by zacjansheski** [comment]
This issue is no longer actual in master
